Tensile/KernelWriterAssembly.py

```diff
@@ -6924,19 +7045,21 @@ def closeSumAtLeastUnroll(self, kernel, prefetch, isOptNLL, isPap, isNGLL):
         kStr += "label_%04u:%s" % (toPGR1, self.endLine)
       else:
         if isOptNLL:
-          endSumLabel = self.getNamedLabel("Summation_End_OptNLL")
-          # If is PAP inside OptNLL: Swap the LRO (if EPS, depends on if BreakAtEvenIter)
-          if self.prefetchAcrossPersistent and isPap:
-            # local read swap offset code
-            kStr += self.getLocalReadSwapOffset(kernel, 1)
-
-            # Jump to Summation
-            kStr += inst("s_branch", "%s"%endSumLabel, "jump to Summation End")
-            kStr += "\n"
-            # Append label for pure OptNLL (no PAP interleaved)
-            kStr += "%s:\n" % self.getNamedLabel("SkipTo_PureOptNLL_LastTile")
-
-          else:
+            endSumLabel = self.getNamedLabel("Summation_End_OptNLL")
+          ## If is PAP inside OptNLL: Swap the LRO (if EPS, depends on if BreakAtEvenIter)
+          #if self.prefetchAcrossPersistent and isPap:
+          #  # in PrefetcGlobalRead=2 case, local read swap is already done in NGLL
+          #  if not kernel["PrefetchGlobalRead"] == 2:
+          #    # local read swap offset code
+          #    kStr += self.getLocalReadSwapOffset(kernel, 1)
+
+          #  # Jump to Summation
+          #  kStr += inst("s_branch", "%s"%endSumLabel, "jump to Summation End")
+          #  kStr += "\n"
+          #  # Append label for pure OptNLL (no PAP interleaved)
+          #  kStr += "%s:\n" % self.getNamedLabel("SkipTo_PureOptNLL_LastTile")
+
+          #else:
```

Comment:
Probably best to delete big chunks of old code, unless it's likely we'll go back to using it soon

Reply:
OK. I will delete this part of code.

Reply:
Deleted.